lib/index.js

```diff
@@ -9,6 +9,16 @@ const defaultConfig = {
 // Check for valid number.
 const isNumber = input => Number(input) === Number(input)
 
+// Add a class to an element.
+const addClass = 'classList' in document.documentElement
+  ? (element, className) => {
+    element.classList.add(className)
+  }
+  : (element, className) => {
+    const oldClass = element.getAttribute('class') || ''
+    element.setAttribute('class', `${oldClass} ${className}`)
+  }
+
```

Comment:
This way the condition is evaluated once instead of every time the function is called.